.github/workflows/purge-test-resources.yaml

```diff
@@ -101,9 +103,9 @@ jobs:
           done
   create_issue_on_failure:
     name: Create issue for failing purge test resources run
-    needs: [purge_gchr_dev, purge_azure_resources]
+    needs: [purge_ghcr_dev, purge_azure_resources]
     runs-on: ubuntu-latest
-    if: always() && (needs.purge_gchr_dev.result == 'failure' || needs.purge_azure_resources.result == 'failure')
+    if: ${{ github.event_name == 'workflow_dispatch' }} && (needs.purge_ghcr_dev.result == 'failure' || needs.purge_azure_resources.result == 'failure')
```

Comment:
don't we want to check `github.event_name == 'schedule'`?

This logic is like: `is workflow dispatch AND ( purge GHCR failed OR purge Azure failed)`

Reply:
This should be `!= workflow_dispatch`. Updated that